### PR TITLE
Store credentials in a csv file

### PR DESCRIPTION
**Pull request by Invorom**
#7

### CODE REVIEW

**Review by opixelum — CHANGES_REQUESTED** [review]
Move storeCredentials function from store files to credentials files & delete store files

File: include/credentials.h [hunk 1]
```diff
@@ -1,4 +1,9 @@
 #include <stdlib.h>
+#include <stdio.h>
+#include "password.h"
+
+char storeCredentials(Credentials_t);
+
 
 typedef struct Credentials_t {
     char *label;
```
Comment (opixelum) [inline include/credentials.h:7]: Put struct above storeCredentials() & remove double blank lines

File: Makefile [hunk 2]
```diff
@@ -59,7 +59,6 @@ $(CREDENTIALS_O): $(CREDENTIALS_C)
 	@mkdir -p build
 	$(CC) $(CFLAGS) -c $(CREDENTIALS_C) -o $@
 
-
```
Comment (opixelum) [inline Makefile]: Add back the blank line

File: include/credentials.h [hunk 3]
```diff
@@ -1,4 +1,9 @@
 #include <stdlib.h>
+#include <stdio.h>
+#include "password.h"
+
+char storeCredentials(Credentials_t);
```
Comment (opixelum) [inline include/credentials.h]: Add doxygen

File: lib/credentials.c [hunk 4]
```diff
@@ -1 +1,29 @@
...
+    FILE *file = fopen("credentials.csv", "r");
+    if (file == NULL)
+    {
+        file = fopen("credentials.csv", "a");
+        fprintf(file, "Label, URL, Username, Email, Password\n");
+    }
+    else
+    {
+        file = fopen("credentials.csv", "a");
+    }
```
Comment (opixelum) [inline lib/credentials.c:20]: Curly braces can be omitted :
```c
else file = fopen("credentials.csv", 'a');
```

**Review by opixelum — CHANGES_REQUESTED** [review]

File: lib/credentials.c (hunk 4 above)
Comment (Invorom) [reply lib/credentials.c:20]: No

**Review by opixelum — APPROVED** [review]
LGTM

**Pull request merged**